src/lib/testLogger.js

```diff
@@ -86,20 +85,15 @@ VerboseLogger.prototype.traverseTestResults = function(node, indentation) {
  * @param {string} indentation - Indentation used for formatting.
  */
 VerboseLogger.prototype.printTestTitles = function(testTitles, indentation) {
-  let prefixColor, statusPrefix;
+  let statusPrefix;
 
   for (let i = 0; i < testTitles.length; i++) {
     if (testTitles[i].failureMessages.length === 0) {
-      prefixColor = colors.GREEN;
-      statusPrefix = this._config.noHighlight ? 'PASS - ' : '\u2713 ';
+      statusPrefix = chalk.green('\u2713 ');
```

Comment:
Note: this part is slightly different. We'll now always print the unicode characters instead of PASS/FAIL. But since colorization is done globally, we won't colorize those characters. I think this is more inline with the config option and doesn't conflate unicode with terminal colors.